src/components/pages/CompensationOption/Footer.js

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import { Link } from "react-router-dom"
+import { Typography, makeStyles, Link as MuiLink } from "@material-ui/core"
+import resourceSections from "data/resourceSections"
+
+const useStyles = makeStyles(theme => ({
+  sectionTitle: {
+    marginBottom: 20,
+    marginTop: 30,
+  },
+}))
+
+const Footer = ({ currentSection, resource, state, slug }) => {
+  const classes = useStyles()
+
+  return (
+    <>
+      <Typography variant="h4" className={classes.sectionTitle}>
+        JUMP TO SECTION
+      </Typography>
+
```

Comment:
TODO: Add a "Next Section" button that just brings you to the next section. This is slightly tricky because not all of the resources have the exact same options, so some refactoring will probably be necessary.